docs/runbooks/CnaoDown.md

```diff
@@ -2,29 +2,31 @@
 
 ## Meaning
 
-Cluster-network-addons-operator (CNAO) deploys additional networking components on top of the cluster.
-This alert fires when CNAO operator is down.
+The `Cluster-network-addons-operator` (CNAO) deploys additional networking components on top of the cluster.
+This alert fires when CNAO is down.
```

Comment:
I think "the CNAO" sounds better. It's your call.

Reply:
IMO this doesn't sound that great in most of the sentences ("check the CNAO's operator pod namespace"), so I'd keep it as is.